<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GMessage</class>
 <widget class="QWidget" name="GMessage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>110</width>
    <height>23</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GMessage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="m_messageLayout" stretch="0,0,0">
     <item>
      <layout class="QVBoxLayout" name="m_iconLayout">
       <item>
        <widget class="QToolButton" name="m_icon">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="m_message">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Message</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="m_closeLayout">
       <item>
        <widget class="QToolButton" name="m_close">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
